watson-developer-cloud/discovery-components
# fix: revert previous facet changes and make new changes

#### What do these changes do/fix?
- revert previous commit (changes were requested by Chris Cook [here](https://ibm-analytics.slack.com/archives/G0117CFBJTU/p1658509360115819?thread_ts=1658332780.760249&cid=G0117CFBJTU))
- re-add double quotes to query
- adjust query decoding to use '::'
- clean up test warnings and logs

Contributes to https://github.ibm.com/Watson-Discovery/disco-issue-tracker/issues/12682

#### How do you test/verify these changes?
Try the following on each of these two [staging](https://us-south.discovery.test.watson.cloud.ibm.com/v2/instances/crn%3Av1%3Astaging%3Apublic%3Adiscovery%3Aus-south%3Aa%2F24f453567f259b67c34ae38a6e6026fb%3A97732895-f1ad-4a2b-9951-548ee712f480%3A%3A/projects/918a1017-e178-4bc8-9b0e-a5950763ac16/workspace) / [local](http://localhost:4000/instances/crn%3Av1%3Astaging%3Apublic%3Adiscovery%3Aus-south%3Aa%2F24f453567f259b67c34ae38a6e6026fb%3A97732895-f1ad-4a2b-9951-548ee712f480%3A%3A/projects/918a1017-e178-4bc8-9b0e-a5950763ac16/workspace) (with this branch and linked to tooling)
1. Run an empty search.
2. Apply a `National Institute of Standards and Technology` facet filter.
3. Note that staging returns nothing, but local returns 56 results.

#### Have you documented your changes (if necessary)?

#### Are there any breaking changes included in this pull request?
